src/components/Admin/Admin.js

```diff
@@ -102,7 +102,7 @@ class Admin extends Component {
 
     return (
       <div>
-        {cookies.get('showAdmin') === 'true' ? (
+        {cookies.get('showAdmin') === 'false' ? (
```

Comment:
Why is this change done? Please explain. Rest PR looks good. Thanks! :-)

Reply:
@Akshat-Jain I was checking for Admin component(which requires access), if its working. Forgot to change it, thanks for pointing out.